learning/web/content/modules/antipsychotics/CON155606_4.mdx

```diff
@@ -7,43 +7,43 @@ import Expander from '../../../src/components/Expander'
 ![prescription pad](../../assets/con146653.jpg)
 The patient should be fully involved in decisions about treatment, including the possibility of psychological, occupational and social therapies. It is important to bear the following points in mind when starting antipsychotic treatment:
 
-* Use antipsychotics within their licensed indications and doses, except on specialist advice  
-  
+* Use antipsychotics within their licensed indications and doses, except on specialist advice
```

Comment:
It might be that GitHub is being funny about showing the diff, but it looks like this list has been reordered? Is there a reason for this? Why did this content need to change?